packages/drivers/routerlicious-driver/src/urlUtils.ts

```diff
@@ -21,30 +21,26 @@ export const replaceDocumentIdInPath = (urlPath: string, documentId: string): st
     urlPath.split("/").slice(0, -1).concat([documentId]).join("/");
 
 export const getDiscoveredFluidResolvedUrl = (resolvedUrl: IFluidResolvedUrl, session: ISession): IFluidResolvedUrl => {
-    if (session) {
```

Comment:
`session` is always true?

Reply:
Double checked, looks correct. Thanks for the cleanup